mappings/net/minecraft/entity/mob/ZoglinEntity.mapping

```diff
@@ -3,6 +3,7 @@ CLASS net/minecraft/class_5136 net/minecraft/entity/mob/ZoglinEntity
 	FIELD field_23732 movementCooldownTicks I
 	FIELD field_23733 USED_MEMORY_MODULES Lcom/google/common/collect/ImmutableList;
 	FIELD field_23734 BABY Lnet/minecraft/class_2940;
+	FIELD field_30511 ATTACK_TARGET_START_TIME I
```

Comment:
```suggestion
	FIELD field_30511 ATTACK_TARGET_DURATION I
```

The `startTime` parameter of `Brain#remember` is incorrect. It's a bad match from [20w10a](https://github.com/FabricMC/yarn/commit/1b7aa91a4ddc43fd259bc4011834e7e5112143b8#diff-d50255fd81242e9db855cd3c554979f54aa7dd47c5c71390990e16c56d431ca5).

Reply:
Turned into an issue. #3399